docs/src/gui/gladevcp.adoc

```diff
@@ -1044,13 +1044,13 @@ Insensitized children will not respond to input.
 
 `hal_hidetable` has one HAL BIT input pin which controls if it's child widgets are hidden or not.
 
-<Panel_basename>.<widgetname>:: FIXME pin name
+`_<Panel_basename>_._<widgetname>_`:: FIXME pin name
   If the pin is low then child widgets are visible which is the default state.
 
 `hal_table` and `hal_hbox` have one HAL BIT input pin which controls if their child widgets are sensitive or not. +
 These widgets's pin name uses the convention:
 
-<Panel_basename>.<widgetname>:: FIXME pin name
+`_<Panel_basename>_._<widgetname>_`:: FIXME pin name
```

Comment:
I am pretty sure this renders as
`_<Panel_basename>_._<widgetname>_`

Reply:
Done.